Makefile

```diff
@@ -108,10 +109,19 @@ run-dev:
 	    --namespace=${NAMESPACE}
 
 # Depends on operator-sdk for now
-.PHONE: debug
+.PHONY: debug
 debug:
 	OPERATOR_NAME=local-operator \
     WATCH_NAMESPACE=operator \
 	dlv debug --headless  --listen=:2345 --headless --log --api-version=2 cmd/manager/main.go -- \
-	--kubeconfig ~/.kube/config --disable-admission-controller
-
+	--kubeconfig ${KUBECONFIG} --disable-admission-controller
+
+.PHONY: reverse-proxy
+reverse-proxy:
```

Comment:
This is cool. I'd rather have it run with a `kubectl run` though instead of having a deployment.

Reply:
Also it doesn't work unless you have a spinnaker-operator service already in there. Maybe we can split the logic between `make setup-reverse-proxy` and `make debug`?

Reply:
I tried with a `kubectl run` but for some reason the deployment kept getting killed after sending that command to the background (and we would now end up with 2 forked processes to manage instead of 1). Also since the pod needs to mount your local ssh public key, the way to specify that in the run command line is with a `--overrides` parameter, which is basically the json spec of the deployment specifying the volume configuration, making the command really long.

Regarding `service` part that is automatically created by the webhook controller setup during startup, there's no need to provision it separately.